pyaedt/edb_core/edb_data/terminals.py

```diff
@@ -476,6 +476,13 @@ class PadstackInstanceTerminal(Terminal):
     def __init__(self, pedb, edb_object):
         super().__init__(pedb, edb_object)
 
+    @property
+    def position(self):
```

Comment:
Just missing the docstring. Otherwise LGTM.

Reply:
oh yes thanks